netbox/extras/tests/test_api.py

```diff
@@ -520,3 +523,76 @@ def test_render_configcontext_for_object(self):
         configcontext6.sites.add(site2)
         rendered_context = device.get_config_context()
         self.assertEqual(rendered_context['bar'], 456)
+
+
+class CreatedUpdatedFilterTest(APITestCase):
+
+    def setUp(self):
+
+        super().setUp()
+
+        self.site1 = Site.objects.create(name='Test Site 1', slug='test-site-1')
+        self.rackgroup1 = RackGroup.objects.create(site=self.site1, name='Test Rack Group 1', slug='test-rack-group-1')
+        self.rackrole1 = RackRole.objects.create(name='Test Rack Role 1', slug='test-rack-role-1', color='ff0000')
+        self.rack1 = Rack.objects.create(
+            site=self.site1, group=self.rackgroup1, role=self.rackrole1, name='Test Rack 1', u_height=42,
+        )
+        self.rack2 = Rack.objects.create(
+            site=self.site1, group=self.rackgroup1, role=self.rackrole1, name='Test Rack 2', u_height=42,
+        )
+
+        # change the created and last_updated of one
+        time.sleep(2)  # to have a difference on the last_updated
```

Comment:
Because tests are run on every merge, they need to execute as fast as reasonably possible. Using `sleep()` here introduces 14 seconds of unnecessary delay (two seconds for each of the seven individual test methods). It also doesn't make sense because you're manually setting an arbitrary create time in the next line.

Instead, you can simply use the `update()` queryset method to assign arbitrary datetimes for the objects' `created` and `last_updated` fields.